src/UI/templates/default/Divider/divider.less

```diff
@@ -0,0 +1,12 @@
+h6.il-divider {
+	padding: @padding-small-vertical @padding-small-horizontal;
+	background-color: darken(@il-secondary-container-bg, 5%);
+	display: block;
+	font-size: @font-size-small;
+	line-height: @line-height-small;
+	color: lighten(@text-color, 10%);
```

Comment:
Do not use lighten in any other files than variables.less. See: https://github.com/ILIAS-eLearning/ILIAS/tree/trunk/templates#colors

Note that this is otherwise extremely hard to track for designers. We had just recent complaints about this. 

Either introduce an il-divider-color or an il-text-color-light, with an according description of where you intend to use this.